samples/react/src/RouteHandler.js

```diff
@@ -97,7 +75,10 @@ class RouteHandler extends React.Component {
           itemId: routeData.sitecore.route.itemId,
           ...routeData.sitecore.context,
         });
-        this.setState({ notFound: false });
+
+        if (this.state.notFound) {
```

Comment:
Should this be the following? Why would we set notFound to false if `notFound === true` ?
```suggestion
        if (!this.state.notFound) {
```

Reply:
So, we want to set state: notFound: false, in case if it's true, so it was correct

Reply:
We have data, and we want to change state

Reply:
Makes sense, thanks for explaining that!